debiandesktop_sg_rules.tf

```
@@ -42,8 +47,8 @@ resource "aws_security_group_rule" "debiandesktop_egress_to_nessus_" {
 # Allow egress to anywhere via HTTP and HTTPS
 # For: Operator web access, package downloads and updates
 resource "aws_security_group_rule" "debiandesktop_egress_to_anywhere_via_allowed_ports" {
+  for_each = toset([for port in ["80", "443"] : port if lookup(var.operations_instance_counts, "debiandesktop", 0) > 0])
```

I think for these guys it is more efficient to do:
```suggestion
  for_each = lookup(var.operations_instance_counts, "debiandesktop", 0) > 0 ? toset(["80", "443"]) : toset([])
```

---

Agreed!  See https://github.com/cisagov/cool-assessment-terraform/pull/84/commits/206d0cbbe3c5979a395edc655349274b20dc2a28.